format/FormatUtils.js

```diff
@@ -27,6 +28,18 @@ export function createRenderer(formatter) {
     };
 }
 
+/**
+ * Generate a formatter that returns a React Element.
+ *
+ * @param {function} [formatter] - an existing formatter method.
+ */
+export function createElFormatter(formatter) {
+    return function(v, opts) {
+        const html = {__html: formatter(v, opts)};
+        return span({dangerouslySetInnerHTML: html});
```

Comment:
Was thinking again as to whether or not we should have any reservations here about using dangerously...  

One argument we can make is that this is displaying the output of our formatters, which currently take primitives and return strings that might have markup, but where the markup is generated by us and can't contain random generated stuff.

Realizing that one exception to that is our support for `tipFn` could end up injecting some additional content into the markup from another source.  But that should probably be escaped anyway - maybe we want to do that for good measure.

Reply:
Doesn't the application data within the *markup* present the risk? (Not sure we need to address this immediately -- don't see how this any worse that what we currently do in sencha hoist, regarding inserting application data in to the Dom